prow/jobs/jinja/presubmits/code_gen_tests.jinja2

```diff
@@ -0,0 +1,58 @@
+  aws-controllers-k8s/code-generator:
+  - name: unit-test
+    decorate: true
+    optional: false
+    always_run: true
+    spec:
+      serviceAccountName: pre-submit-service-account
+      containers:
+      - image: public.ecr.aws/m5q3e4b2/prow:prow-unit-0.0.1
+        resources:
+          limits:
+            cpu: 1
+        command: ["make", "test"]
+
+{% for service in code_gen_presubmit_services  %}
+  - name: {{ service }}-controller-test
+    decorate: true
+    optional: true
+    always_run: true
+    labels:
+      preset-dind-enabled: "true"
+      preset-kind-volume-mounts: "true"
+    extra_refs:
+    - org: aws-controllers-k8s
+      repo: runtime
+      base_ref: main
+      workdir: false
+    - org: aws-controllers-k8s
+      repo: test-infra
+      base_ref: main
+      workdir: true
+    - org: aws-controllers-k8s
+      repo: {{ service }}-controller
+      base_ref: main
+      workdir: true
+    spec:
+      serviceAccountName: pre-submit-service-account
+      containers:
+      - image: public.ecr.aws/m5q3e4b2/prow:prow-integration-0.0.4
+        securityContext:
+          privileged: true
+        resources:
+          requests:
+            cpu: 2
+          limits:
+            cpu: 8
```

Comment:
Can definitely remove this. These requests and limits are arbitrary at best, misleading at worst. I will probably remove them from the other e2e tests.